src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -251,6 +251,14 @@ public static String formatDateTimeForInstructorHomePage(Date date) {
         return sdf.format(date);
     }
 
+    public static String formatDateTimeForFeedbackSubmissionEditPage(Date date, double timeZone) {
+        StringBuffer formattedDateTime = new StringBuffer(formatTime12H(date));
+        formattedDateTime.append(" (")
+                         .append(getTimeZoneFromDoubleOffset(timeZone).getID().replace("GMT", "UTC "))
```

Comment:
Check out `TimeHelper`'s `formatDateTimeForComments` for an example how to use the `TimeZone` object instead.